modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/constants/ObjectDefinitionsScreenNavigationEntryConstants.java

```diff
@@ -48,6 +50,8 @@ public class ObjectDefinitionsScreenNavigationEntryConstants {
 
 	public static final String ENTRY_KEY_RELATIONSHIPS = "relationships";
 
+	public static final String ENTRY_KEY_STATES = "states";
```

Comment:
Hi, @cgmonte this constant seems too generic to me. What do you think of replacing it with something like this:
`ENTRY_KEY_STATES_MANAGER = "states-manager";`